css/main.css

```diff
@@ -0,0 +1,212 @@
+  outline: none;
+  margin: 1.5em 0em;
+  cursor: pointer; }
+
+input[type="email"],
+input[type="password"],
+input[type="text"] {
+  border-radius: 0.5em;
+  padding: 0.8em 0em;
+  padding-left: 0.8em;
+  width: 100%;
+  border: 1px solid #1b1242;
+  font-size: 1em;
+  margin-bottom: 0.8em;
+  margin-right: 0.8em;
+  background-color: #e8f0fe; }
+  input[type="email"]:focus,
+  input[type="password"]:focus,
+  input[type="text"]:focus {
+    background-color: #fff;
+    outline: none; }
+  input[type="email"]::-webkit-input-placeholder,
+  input[type="password"]::-webkit-input-placeholder,
+  input[type="text"]::-webkit-input-placeholder {
+    color: rgba(27, 18, 66, 0.5); }
+
+.col-md-6 {
+  width: 50%; }
+
+.row {
+  display: flex;
+  flex-direction: row; }
+
+.container {
+  display: flex;
+  flex-direction: column; }
+
+@media (min-width: 768px) {
+  .hide-md {
+    display: none !important; } }
+@media (max-width: 767px) {
+  .col-sm-12 {
+    width: 100%; }
+
+  .hide-sm {
+    display: none; } }
+/* COMPONENTS ------------------------------------------- */
+.auth-left {
+  background-color: #1b1242;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-height: 100vh;
+  justify-content: center;
+  color: #e8f0fe;
+  font-family: Arial; }
+  .auth-left p {
+    text-align: left; }
+  .auth-left button {
+    background-color: rgba(0, 0, 0, 0);
+    border: 2px solid #e8f0fe;
+    color: #e8f0fe; }
+.auth-right {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-height: 100vh;
+  justify-content: center; }
+  .auth-right .pg-title {
+    display: flex;
+    justify-content: center;
+    color: #1b1242;
+    font-weight: bold;
+    text-align: center; }
+  .auth-right input[type="button"] {
+    background-color: rgba(0, 0, 0, 0); }
+    .auth-right input[type="button"]:hover {
+      background-color: #1b1242;
+      color: #e8f0fe; }
+  .auth-right p.password-reset {
+    float: right; }
+
+.socials {
+  display: flex;
+  justify-content: center; }
+
+.icon {
+  display: flex;
+  justify-content: center;
+  border-radius: 50%;
+  border: 1px solid #1b1242;
+  background-color: #e8f0fe;
+  width: 3em;
+  height: 3em;
+  margin: 5px;
+  cursor: pointer; }
+  .icon:hover {
+    background-color: #fff; }
+
+.logo {
+  display: flex;
+  justify-content: center; }
+
+img.logo {
+  width: 6em;
+  height: 6em; }
+
+.auth-link {
+  display: flex;
+  justify-content: center; }
+  .auth-link p {
+    color: rgba(27, 18, 66, 0.5); }
+
+/* devices = laptops, desktops, tablets ---------------*/
+@media (min-width: 1025px) {
+  .auth-left {
+    padding-top: 4em; }
+    .auth-left p {
+      width: 50%;
+      font-size: 1.5em;
+      margin: 1.4em 0em 0.6em 0em; }
+    .auth-left button {
+      width: 55%; }
+  .auth-right .pg-title {
+    font-size: 1.5em;
+    margin: 0.5em 0em; }
+  .auth .auth-form {
+    max-width: 60%; } }
+/*devices = Tablets, iPads-------------*/
```

Comment:
What do you feel about putting space between the comment line and the code?
I would suggest that it would be to leave space between. The work looks great indeed.

Reply:
That is a transpired file, we do have to change or edit something inside there. Unless you have some configuration we can add to make it look like a real human written document.